MetriCam2_SDK.sln

```diff
@@ -99,274 +105,452 @@ Global
+		{03558646-61B1-4F4E-BF70-7A623DF27095}.ReleaseStrongName|Any CPU.Build.0 = ReleaseStrongName|Any CPU
+		{03558646-61B1-4F4E-BF70-7A623DF27095}.ReleaseStrongName|x64.ActiveCfg = ReleaseStrongName|Any CPU
+		{03558646-61B1-4F4E-BF70-7A623DF27095}.ReleaseStrongName|x64.Build.0 = ReleaseStrongName|Any CPU
+		{03558646-61B1-4F4E-BF70-7A623DF27095}.ReleaseStrongName|x86.ActiveCfg = ReleaseStrongName|Any CPU
+		{03558646-61B1-4F4E-BF70-7A623DF27095}.ReleaseStrongName|x86.Build.0 = ReleaseStrongName|Any CPU
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Debug|Any CPU.ActiveCfg = Debug|x86
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Debug|x64.ActiveCfg = Debug|x64
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Debug|x64.Build.0 = Debug|x64
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Debug|x86.ActiveCfg = Debug|x86
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Debug|x86.Build.0 = Debug|x86
+		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.DebugStrongName|Any CPU.ActiveCfg = Debug|x86
+		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.DebugStrongName|x64.ActiveCfg = Debug|x64
+		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.DebugStrongName|x86.ActiveCfg = Debug|x86
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Release|Any CPU.ActiveCfg = Release|x86
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Release|x64.ActiveCfg = Release|x64
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Release|x64.Build.0 = Release|x64
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Release|x86.ActiveCfg = Release|x86
 		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.Release|x86.Build.0 = Release|x86
+		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.ReleaseStrongName|Any CPU.ActiveCfg = Release|x86
+		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.ReleaseStrongName|x64.ActiveCfg = Release|x64
+		{DE24F649-A2A1-47E7-9C63-F032946C0C2F}.ReleaseStrongName|x86.ActiveCfg = Release|x86
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Debug|Any CPU.ActiveCfg = Debug|x86
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Debug|x64.ActiveCfg = Debug|x64
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Debug|x64.Build.0 = Debug|x64
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Debug|x86.ActiveCfg = Debug|x86
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Debug|x86.Build.0 = Debug|x86
+		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.DebugStrongName|Any CPU.ActiveCfg = Debug|x86
+		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.DebugStrongName|x64.ActiveCfg = Debug|x64
+		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.DebugStrongName|x86.ActiveCfg = Debug|x86
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Release|Any CPU.ActiveCfg = Release|x86
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Release|x64.ActiveCfg = Release|x64
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Release|x64.Build.0 = Release|x64
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Release|x86.ActiveCfg = Release|x86
 		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.Release|x86.Build.0 = Release|x86
+		{177E27F4-7C6E-4BF7-BDE2-97888EE2682C}.ReleaseStrongName|Any CPU.ActiveCfg = Release|x86
```

Comment:
I think one must assign a platfom/configuration combi for each platform/configuration combination allowed by the solution. Since a lot of projects do not support AnyCPU builds, one must assign another platform. But this should not be an issue, because the respective projects are disabled for AnyCPU solutions builds anyway.

Reply:
Okay. I would have expected that disabled projects would have no entry.